option.go

```diff
@@ -153,6 +153,8 @@ func (l *Literal) parse(p *Parser) error {
 type NamedLiteral struct {
 	*Literal
 	Name string
+	// PrintsColon is true when the Name must be printed with a colon suffix
+	PrintsColon bool
```

Comment:
Where is this tested?

In previous versions, was the colon printed automatically/was it part of the Name? If so, this is backwards incompatible.

Reply:
the colon was never part of the key. this change preserves that information such that any printer/formatter could use it to stay close to the original source.